### ISSUE TITLE
Use strict error when bundling

### ISSUE THREAD

**Issue by Torinth** [novus/nvd3#1858]
Hi, I am using the latest 1.8.4 version.  If I try to minify the file it tells me on like 5072 and 5073 that it cant minify with strict on due to undefined variable getX and getY.  I saw some previous issues relating to this but it looks like this spot is missed.  Temporarilly I have remove the use strict from I think like 5056 and it lets me minify

**Comment by liquidpele** [comment]
Check latest master, I remember something about those methods so it might be fixed in latest.   Other than that, if you fix it send us a pull request :)

**Comment by Torinth** [comment]
unfortunately my fix is to remove use strict from that one spot.  I doubt yall would want that as a pull request.

We are using npm to bring this package in.  We don't want auto updates and have locked down all of our versions so I'm not sure I could use master in that way?

**Comment by liquidpele** [comment]
I'm not saying to use master (that's dev, don't do that), I'm saying that if you want to try and fix the project to work with strict on then you would first want to make sure that it's not already fixed in the latest code.

**Comment by Torinth** [comment]
Yeah sorry I don't actually know what the fix is.  I'm not sure what or where something needs to be added to fix this.  just bringing up an issue I found.  It also turns out we cant fix this locally because our build server pulls npm packages at build time

**Comment by Torinth** [comment]
Ok so I looked at the built file and tested something and it looks like it works (I did not look at the non built files).  At line 4967 add right below it and it will fix the problem

, getX
, getY
